examples/stock_example_parallel.jl

```diff
@@ -0,0 +1,97 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Solving an decision hazard stock problem using multiprocessed dynamic programming :
+# We decide which quantity to store before the randomness happens
+# If too much energy comes, the excess is wasted
```

Comment:
It is not necessary to speak of energy but it is a natural problem in energy as we don't want to waste renewables but cannot take the decision after the randomness realization